generators/app/index.js

```diff
@@ -1,14 +1,14 @@
-// modules used by the generator
-const generator = require('yeoman-generator');
 const chalk = require('chalk');
-const packagejs = require('../../package.json'); // gives access to the package.json data
+const fs = require('fs');
+const Generator = require('yeoman-generator');
```

Comment:
Because it's a class now, so it's Classy. I actually followed JHipster's conventi